content/en-us/_index.md

```diff
@@ -4,14 +4,14 @@ title: "Home"
+Many IoT designs now being used globally are well-worn and time-tested, which often come from mature domains such as *sensor networks*, *[control system](https://en.wikipedia.org/wiki/Control_system)* architectures, *[machine-to-machine ](https://en.wikipedia.org/wiki/Machine_to_machine)* networks, and *[embedded system](https://en.wikipedia.org/wiki/Embedded_system)* development. However, the lack of visibility of those mature designs is causing many people to [reinvent the wheel](https://en.wikipedia.org/wiki/Reinventing_the_wheel) for their solution, when they would prefer not. To further complicate the matter for experienced practitioners, even when a long-standing design is known, the design often needs to be revised to take into account cloud development concepts. 
 
-The IoT Atlas is a resource for both new and long-time builders of internet of things solutions. It updates and expands upon well-worn designs with the presumption that today, building "Internet of Things" solutions almost always means that "a hyper-scale cloud is available" to practitioners building each solution. Taking the stance that solutions development can rely upon the existence of a hyper-scale cloud does not imply the IoT Atlas' designs must be connected in order to operate. Instead the designs attempt to leverage the strength of a fleet of devices operating locally in remote environments for long periods of time **and** the cloud's strengths of massive scale, global ubiquity, and solution agility. The designs can then be used to build bar-raising solutions that expect intermittent, long-duration, or primarily-disconnected network outages as a fundamental influence of their designed components. Essentially the designs herein follow the IoT philosophy that *the edge completes the cloud*. 
+The IoT Atlas is a resource for both new and long-time builders of IoT solutions. It updates and expands upon well-worn designs with the presumption that today, building "Internet of Things" solutions almost always means that "a hyper-scale cloud is available" to practitioners building each solution. Taking the stance that solutions development can rely upon the existence of a hyper-scale cloud does not imply the IoT Atlas' designs must be connected in order to operate. Instead, the designs attempt to leverage the strength of a fleet of devices operating locally in remote environments for long periods of time **and** the cloud's strengths of massive scale, global ubiquity, and solution agility. The designs can then be used to build bar-raising solutions that expect intermittent, long-duration, or primarily-disconnected network outages as a fundamental influence of their designed components. Essentially the designs herein follow the IoT philosophy that *the edge completes the cloud*. 
```

Comment:
This propagation of "IoT" in place of "internet of things" aligns. Thanks for the change.